lib/index.js

```diff
@@ -198,8 +199,14 @@ function walk (opts, nodes) {
 
     // if not, we have an object, so we need to run the attributes and contents
     if (node.attrs) {
+      // adapt regex to work with attribute values
+      ignoredExpression = new RegExp(`.*@${ignoredBefore}(.+?)${ignoredAfter}`)
```

Comment:
I think it’s better to put on line 154

`ignoredExpression` I think it's worth renaming to `ignoredAttrsExpression`